docs/deploy/upload.py

```diff
@@ -0,0 +1,167 @@
+import argparse
+import hashlib
+import mimetypes
+import os
+import pathlib
+
+import boto3
+
+HERE = pathlib.Path(__file__).parent
+
+if __name__ == "__main__":
+
+    def dir_path(string):
+        if os.path.isdir(string):
+            return string
+        else:
+            raise NotADirectoryError(string)
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--preview",
+        action="store_true",
+        default=True,
+        help="whether the upload should go under the short-lived /previews path",
+    )
+    parser.add_argument(
+        "--version-file",
+        type=str,
+        default=HERE / ".." / ".." / "VERSION",
```

Comment:
Does this syntax actually work? I've never seen it, and Python3 on my phone complains about dividing strings by each other.

Reply:
`HERE` is a Path, not a string

Reply:
Ah, that makes more sense. 😂